test/sdn_epc/policymaker_test.exs

```diff
@@ -0,0 +1,82 @@
+defmodule SdnEpc.PolicymakerTest do
+  use ExUnit.Case, async: false
+
+  @count_duration Application.get_env(:sdn_epc, :pic_count_duration)
+  @msgs_limit Application.get_env(:sdn_epc, :pic_msgs_limit)
+  @drop_duration Application.get_env(:sdn_epc, :pic_drop_duration)
+
+  setup do
+    Application.stop(:sdn_epc)
+    :ok = Application.start(:sdn_epc)
+  end
+
+  test "detect DDos" do
+    # GIVEN
+    msg = OfpMessage.get(:packet_in)
+    receive_timeout = SdnEpc.TestHelper.calculate_receive_timeout(@drop_duration)
+    datapath_id = '01'
+
+    # WHEN
+    simulate_ddos(msg)
+    SdnEpc.Forwarder.send_msg_to_controller({datapath_id, self()}, msg)
+
+    # THEN
+    refute_receive(^datapath_id, receive_timeout)
+  end
+
+  test "change blocking_time to false after drop duration
+    (when DDoS occured)" do
+    # GIVEN
+    msg = OfpMessage.get(:packet_in)
+    drop_duration_timeout = @drop_duration * 1000 + 1100
+    datapath_id = '01'
+
+    # WHEN
+    simulate_ddos(msg)
+    Process.sleep(drop_duration_timeout)
+    SdnEpc.Forwarder.send_msg_to_controller({datapath_id, self()}, msg)
+
+    # THEN
+    assert_receive(^datapath_id)
+  end
+
+  test "kill Forwarder after switch mode call timeout" do
```

Comment:
@mentels, Could you help me with that test? It dosen't pass. Could you give me some advice how to write it properly?